### DISCUSSION TITLE
Make BarHitErrorMeter reversible

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#17371] [Ideas] [answered]
Some of my friends ask for having the possibility to make early at right and late at left.
I think it can be cool to have the option in the settings
![image](https://user-images.githubusercontent.com/50368878/159171985-39f28080-fd92-49ad-8b1f-3a1af30e3a70.png)
![image](https://user-images.githubusercontent.com/50368878/159172192-bb0b2549-5501-4e5c-acb7-7c969a4ebf1d.png)

**Comment by peppy** [comment, marked as answer]
Use the rotate or flip controls to do this. Flip doesn't work perfectly yet but rotate will.
